### ISSUE TITLE
Lighter weight or more hinting?

### ISSUE THREAD

**Issue by bluss** [belluzj/fantasque-sans#71]
Hi this is what the font looks like in my configuration.

Linux, XFCE, Fantasque Sans Mono, Size: 11

It looks the same on any hinting setting of Light, Medium, Full.

I think the shapes of the font are great, the accentuation of `[ ( < - +` are great for coding. I feel like this font is just a touch too heavy weight. I'm no font expert, but it may be that it doesn't respond to hinting that much. Either that, or maybe it could be made in a bit lighter version? But not as thin as a "light" font variant normally is.

![screenshot](http://i.imgur.com/Pte4Sj4.png)

**Comment by belluzj** [comment]
> this font is just a touch too heavy weight

Yes, it may be true. However, I don't think I will ever produce a light version of this font, because I did not build it to allow interpolation, and it would be too much work to draw a whole light weight. So I won't fix that, sorry.

> it doesn't respond to hinting that much

Did you install the TTF or OTF version? The TTF includes hinting instructions, so it perhaps ignores system-wide hinting settings. You should try with both versions to see whether they behave the same way in that regard.

**Comment by bluss** [comment]
Thanks for the information. I had the TTF version before. OTF looks mostly the same to me. I'll upload a comparison screenshot. I imagine maybe that's useful to you, the OTF looks worse if I really stare at it (comparing individual letters, still Light hinting setting). Lowercase f and p look worse in OTF.

![screenshot](http://i.imgur.com/qSr7caD.png)

both screenshots look too blurry in firefox, need to view them with anoter viewer.

**Comment by bluss** [comment]
But OTF looks a little little bit lighter. I hope you agree, I might just be seeing things.

**Comment by belluzj** [comment]
Oh my, the second screenshot you sent feels like the individual letters are drunk but still trying to look normal. The lines of code look like waves :)

I believe what you're seeing is: in the TTF version, the hinting is very subtle and the intended shapes of the letters are preserved, but it makes the rendering blurry. For example if a stroke is inbetween two pixels, it spills over the two pixels to stay true to the original, and it looks a bit fatter since it spans two pixels. In the second screenshot, hinting is much more aggressive and the letter shapes are distorted (badly so) to fit on the pixel grid, and overall the rendering is less blurry and the strokes look thinner.
